### ISSUE TITLE
support Calendar and Date

### ISSUE THREAD

**Issue by guilhermesilveira** [caelum/vraptor-dash#17]
support calendar and date, printing its date and hour, english format

**Comment by josedonizetti** [comment]
where it should be supported? for statements, only?

**Comment by guilhermesilveira** [comment]
yep. as statements results they should be printed.in english format

Guilherme Silveira
On Aug 21, 2011 10:49 PM, "josedonizetti" <
reply@reply.github.com>
wrote:
